catalog/admin/module/catalog-admin-module-maplayers/src/main/webapp/map-layers/options.json

```diff
@@ -0,0 +1,85 @@
+{
+  "OSM": {
+    "label": "Open Street Map",
+    "help": {
+      "ol": "https://openlayers.org/en/latest/apidoc/ol.source.OSM.html",
+      "cesium": "https://cesiumjs.org/Cesium/Build/Documentation/createOpenStreetMapImageryProvider.html"
+    },
+    "config": {}
+  },
+  "WMS": {
+    "label": "Web Map Service",
+    "help": {
+      "ol": "https://openlayers.org/en/latest/apidoc/ol.source.ImageWMS.html",
+      "cesium": "https://cesiumjs.org/Cesium/Build/Documentation/WebMapServiceImageryProvider.html"
+    },
+    "config": {
+      "parameters": {
+        "LAYERS": [],
+        "STYLES": "",
+        "VERSION": "1.3.0"
+      }
+    }
+  },
+  "WMT": {
+    "label": "Web Map Tile Service",
+    "help": {
+      "ol": "https://openlayers.org/en/latest/apidoc/ol.source.WMTS.html",
+      "cesium": "https://cesiumjs.org/Cesium/Build/Documentation/WebMapTileServiceImageryProvider.html"
+    },
+    "config": {
+      "matrixSet": "",
+      "layer": "",
+      "format": "image/jpeg",
+      "tileWidth": 256,
+      "tileHeight": 256
+    }
+  },
+  "AGM": {
+    "label": "ArcGIS Map Server",
+    "help": {
+      "ol": "https://openlayers.org/en/latest/apidoc/ol.source.XYZ.html",
+      "cesium": "https://cesiumjs.org/Cesium/Build/Documentation/ArcGisMapServerImageryProvider.html"
+    },
+    "config": {}
+  },
+  "SI": {
+    "label": "Single Tile",
+    "help": {
+      "ol": "https://openlayers.org/en/latest/apidoc/ol.source.ImageStatic.html",
+      "cesium": "https://cesiumjs.org/Cesium/Build/Documentation/SingleTileImageryProvider.html"
+    },
+    "config": {
+      "parameters": {}
+    }
+  },
+  "BM": {
+    "label": "Bing Maps",
+    "help": {
+      "ol": "https://openlayers.org/en/latest/apidoc/ol.source.BingMaps.html",
+      "cesium": "https://cesiumjs.org/Cesium/Build/Documentation/BingMapsImageryProvider.html"
+    },
+    "config": {}
```

Comment:
A Bing Maps key is needed for both OL and Cesium.  This is probably also true for Map Quest and Google Earth.  We might want to include the defaults to remind users they need a key to use these types of layers.